en/advanced-gimbal-control.md

```diff
@@ -1,20 +1,24 @@
 # Gimbal Control Setup
 
-PX4 contains a generic mount/gimbal control driver with different input and output methods. Any input can be selected to drive any output.
-
-First, make sure the driver runs, using `vmount start`, then configure its parameters.
+PX4 contains a generic mount/gimbal control driver with different input and
+output methods. Any input method can be selected to drive any output.
 
 ## Parameters
-The parameters are described in [src/drivers/vmount/vmount_params.c](https://github.com/PX4/Firmware/blob/master/src/drivers/vmount/vmount_params.c). The most important ones are the input (`MNT_MODE_IN`) and the output (`MNT_MODE_OUT`) mode. By default, the input is disabled. Any input method can be selected to drive any of the available outputs.
```

Comment:
So it looks like you need to set MNT_MODE_IN because it is disabled by default and then reboot. Why? You don't say. Does this mean that for any gimbal use you need to set MNT_MOUNT_IN? Similarly MNT_MOUNT_OUT, why is that needed. 

Also we've jumped into parameters here, and then we go to "Configure the gimbal mixer for AUX output". However there is no "context" at the point you talk about parameters. Before you talk about params and config for aux output, it is worth a sentence saying "how everything fits together". Something like "The input/output mixer is configured using a specific file XXXX, using parameters to define the inputs and outputs blah blah blah".